src/wp-includes/class-wp-locale-switcher.php

```diff
@@ -196,10 +196,6 @@ private function load_translations( $locale ) {
 		load_default_textdomain( $locale );
 
 		foreach ( $domains as $domain ) {
-			if ( 'default' === $domain ) {
-				continue;
-			}
-
```

Comment:
Why this? Should the `load_default_textdomain()` call be removed too? Is this the reason why we have to manually call `get_translations_for_domain()`.

Reply:
Hmm yeah at first glance seems odd to remove the `'default' === $domain` check